src/tetris/field/field.ts

```diff
@@ -223,7 +260,7 @@ export default class Field {
 	}
 
 	private _increaseScore(deletedRows: number): void {
-		this._score += deletedRows * 100;
+		this._score += deletedRows * 1000;
```

Comment:
Magic number. I would suggest to put it into a separate config or, at least, introduce a constant, defined at the top of this file.